### ISSUE TITLE
False positive in Rails/FilePath

### ISSUE THREAD

**Issue by bquorning** [bbatsov/rubocop#5615]
On master branch (6c3ea4753bbb9f1ffb0f2ee60400fd0b432ede3f) I’m seeing `Rails/FilePath` complain about this code:

```rb
<<-FOO
  1
  /
FOO
```

```
test.rb:1:1: C: Rails/FilePath: Please use Rails.root.join('path', 'to') instead.
<<-FOO
^^^^^^
```

With `git bisect` I found that the false positive was introduced in 707c9a85c1d99bfee8a10ac404abf6a14d6a88f0 – cc @koic

--------

## Expected behavior

No offense should be found.

## Actual behavior

An offense found for `Rails/FilePath`.

## Steps to reproduce the problem

See above.

## RuboCop version

```
$ rubocop -V
0.52.1 (using Parser 2.5.0.2, running on ruby 2.4.2 x86_64-darwin15)
$ git rev-parse @
6c3ea4753bbb9f1ffb0f2ee60400fd0b432ede3f
```

**Comment by koic** [comment]
Thanks for the feedback.
I reproduced this false positive. I will investigate.

**Comment by bquorning** [comment]
I just found another false positive which may be related.
```rb
'' \
'/'
```
```
test.rb:1:1: C: Rails/FilePath: Please use Rails.root.join('path', 'to') instead.
'' \
^^^
```

**Comment by bquorning** [comment]
One more:
```rb
"#{42}/"
```
```
test.rb:1:1: C: Rails/FilePath: Please use Rails.root.join('path', 'to') instead.
"#{42}/"
^^^^^^^^
```
